sample-app/src/app/index.ts

```diff
@@ -11,20 +11,10 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
+import { Application } from 'lisk-sdk';
 
-'use strict';
+export const createApp = (genesisBlock: JSON, config: JSON): Application => {
```

Comment:
```typescript
export const createApp = (genesisBlock: JSON, config: JSON): Application => new Application(genesisBlock, config)
```

Reply:
We could do this, but i shouldve added the comment to expand the module/custom transaction here